### ISSUE TITLE
Interpolate not working

### ISSUE THREAD

**Issue by BilalReffas** [marmelroy/Interpolate#16]
Hi ! :)
I have the problem that my code below is not printing any values. I'm using Xcode 8 and Swift 3.
```

let i = Interpolate(from: 100.0, to: 3.0, apply: { n in print(n) })
        i.animate(duration: 3.0)
```

**Comment by marmelroy** [comment]
Hi @BilalReffas ! 

This was a bug that was fixed with Interpolate 0.3.1 (just checked again). 

See:
https://github.com/marmelroy/Interpolate/issues/15

Please update.

**Comment by BilalReffas** [comment]
Thank you it work's :)
